src/System.Data.SqlClient/tests/StressTests/System.Data.SqlClient.Stress.Tests/NetUtils.cs

```diff
@@ -216,7 +216,7 @@ public static SqlConnectionStringBuilder GetMultiSubnetFailoverConnectionString(
         private const int MaxUserPortDefaultValueAfterVista = 16384;
 
         private const string TcpTimedWaitDelayName = @"TcpTimedWaitDelay";
-        // according to http://msdn.microsoft.com/en-us/library/ms819739.aspx and http://msdn.microsoft.com/en-us/library/ee377084(BTS.10).aspx, 
+        // according to https://www.microsoft.com/en-us/download/details.aspx?id=56498, 
```

Comment:
Linking to the download doesn't help much, it doesn't tell you what you're looking for in the download. Unfortunately i couldn't work out what it was originally refering to either, i considered changing it to.

```

        private const string TcpTimedWaitDelayName = @"TcpTimedWaitDelay";
		// according to http://msdn.microsoft.com/en-us/library/ms819739.aspx and http://msdn.microsoft.com/en-us/library/ee377084(BTS.10).aspx, 
		// the default value of tcp timed wait delay is 4 minutes
		// from and after Windows Server 2008 this value has been changed to 2 minutes, https://docs.microsoft.com/en-us/biztalk/technical-guides/settings-that-can-be-modified-to-improve-network-performance
		// to stay on the safe side, assuming that the max wait time is still 4 minutes
		private const int TcpTimedWaitDelayDefaultValue = 240; // 4 minutes
```

Reply:
This block of constants (`TcpParametersKeyInRegistry`, `MaxUserPortDefaultValue*`, `TcpTimedWaitDelay`) looks unused. You can just delete them.

Reply:
#32722